app-emulation/winetricks/winetricks-20170614.ebuild

```diff
@@ -21,7 +21,7 @@ SRC_URI="${SRC_URI}
 	kde? ( https://dev.gentoo.org/~tetromino/distfiles/wine/${wtg}.tar.bz2 )"
 
 DESCRIPTION="Easy way to install DLLs needed to work around problems in Wine"
-HOMEPAGE="http://winetricks.org http://wiki.winehq.org/winetricks"
```

Comment:
It's normal to include extra pages if they provide useful documentation. Not for me to judge whether this one qualifies though.